File: 3-Liskov-Substitution-Principle (LSP)/js/LSP-bad-example.js
class Post {
    constructor(content) {
      this.content = content;
    }
  
    display() {
      // Logic to display the post
    }
  }
  
  class ImagePost extends Post {
    constructor(content, imageUrl) {
      super(content);
      this.imageUrl = imageUrl;
    }
  
    // No display method overridden here
  }

File: 3-Liskov-Substitution-Principle (LSP)/python/LSP-bad-example.py
class Post:
    def __init__(self, content):
        self.content = content

    def display(self):
        # Logic to display the post
        pass

class ImagePost(Post):
    def __init__(self, content, image_url):
        super().__init__(content)
        self.image_url = image_url